lib/action_subscriber/middleware/env.rb

```diff
@@ -30,7 +30,7 @@ def initialize(subscriber, encoded_payload, properties)
         @encoded_payload = encoded_payload
         @exchange = properties.fetch(:exchange)
         @headers = properties.fetch(:headers) || {}
-        @message_id = properties.fetch(:message_id)
+        @message_id = properties.fetch(:message_id) || ::Random.new.bytes(3).unpack("H*")[0]
```

Comment:
just for giggles I ran the benchmark ... SecureRandom.hex is 5x faster than just Random.new.bytes(10) (with no unpack executed) 

don't know what it is doing, but looks like SecureRandom is "better"